owncloudApp/src/main/java/com/owncloud/android/dependecyinjection/UseCaseModule.kt

```diff
@@ -27,8 +27,12 @@ import com.owncloud.android.domain.capabilities.usecases.GetCapabilitiesAsLiveDa
 import com.owncloud.android.domain.capabilities.usecases.GetStoredCapabilitiesUseCase
 import com.owncloud.android.domain.capabilities.usecases.RefreshCapabilitiesFromServerAsyncUseCase
 import com.owncloud.android.domain.files.usecases.CreateFolderAsyncUseCase
+<<<<<<< HEAD
```

Comment:
Make sure to remove these before a commit.
Please use 
`git diff HEAD` before committing.

Reply:
I know, rebase problems 😢  It was fixed in next commit